tests/events/modules.kconfig.txt

```diff
@@ -0,0 +1,41 @@
+=== [ATTENTION] Testing Kconfig dependency modelling ===
+=== [ATTENTION] Testing Kconfig dependency modelling ===
+make: Nothing to be done for 'clean'.
+auto_init
+auto_init_xtimer
+board
+core
+core_init
+core_msg
+core_panic
+core_thread_flags
+cortexm_common
+cortexm_common_periph
+cpu
+div
+event
+event_callback
+event_timeout
+isrpipe
+malloc_thread_safe
+newlib
+newlib_nano
+newlib_syscalls_default
+periph
+periph_common
+periph_gpio
+periph_init
+periph_init_gpio
+periph_init_pm
+periph_init_timer
+periph_init_uart
+periph_pm
+periph_timer
+periph_uart
+stdin
+stdio_uart
+stdio_uart_rx
+sys
+test_utils_interactive_sync
+tsrb
+xtimer
```

Comment:
This looks like a debugging artifact to me

Reply:
Thanks!